domain_events/django/apps.py

```diff
@@ -0,0 +1,16 @@
+from django.apps import AppConfig
+from django.conf import settings as djsettings
+
+from domain_events import settings
+
+
+class DomainEventsConfig(AppConfig):
+    name = 'domain_events.django'
```

Comment:
We might want to add an explicit [label](https://docs.djangoproject.com/en/1.11/ref/applications/#django.apps.AppConfig.label) as well.